dashboard/app/controllers/api/v1/pd/workshop_survey_foorm_report_controller.rb

```diff
@@ -25,6 +25,10 @@ def csv_survey_report
       form_version = params[:version]
       workshop_id = params[:workshop_id]
       ws_submissions = Pd::WorkshopSurveyFoormSubmission.where(pd_workshop_id: workshop_id)
+      user_submission_count = ws_submissions.distinct.count(:user_id)
+      if user_submission_count < Pd::SharedWorkshopConstants::MIN_SURVEY_RESPONSE_COUNT
+        return render json: {error: 'Insufficient data to generate a report.'}, status: :unprocessable_entity
```

Comment:
Should we tell them what the minimum survey response count is needed so they know what they need to hit before being able to create the report?

Reply:
Oh gotcha, my bad I missed that it was in the view itself. Then it's good either way!